iceoryx_posh/include/iceoryx_posh/internal/runtime/posh_runtime_impl.hpp

```diff
@@ -44,14 +44,15 @@ class PoshRuntimeImpl : public PoshRuntime
     virtual ~PoshRuntimeImpl() noexcept;
 
     /// @copydoc PoshRuntime::findService
-    cxx::expected<InstanceContainer, FindServiceError> findService(const capro::IdString_t& service,
-                                                                   const capro::IdString_t& instance) noexcept override;
+    cxx::expected<InstanceContainer, FindServiceError>
+    findService(const cxx::variant<Any_t, capro::IdString_t> service,
```

Comment:
We should be more explicit here I think. Something like `AnyIdString`. I also admit that I would strive not to use the `_t` suffix in types visible to the user, only internal ones. Personally I try to restrict it for low level types (primitive types, pointers, ...) and do not use it for classes.

Reply:
That's sound like something we should finally bring on the table in the next developer meetup. We are still using various different naming conventions in iceoryx.